clients/cli/tests/test.rs

```diff
@@ -55,11 +53,10 @@ async fn setup(raise_minimum_delegation: bool, initialize_pool: bool) -> Env {
     // start test validator
     let (validator, payer) = start_validator(raise_minimum_delegation).await;
 
-    // make clients
-    let rpc_client = Arc::new(validator.get_async_rpc_client());
-    let program_client: PClient = Arc::new(ProgramRpcClient::new(
-        rpc_client.clone(),
-        ProgramRpcClientSendTransaction,
+    // make client
+    let rpc_client = Arc::new(RpcClient::new_with_commitment(
+        validator.rpc_url(),
+        CommitmentConfig::confirmed(),
     ));
```

Comment:
removed `ProgramClient` from tests just because we dont need it. planning on dropping token client from the cli itself when i do the 3.0 upgrade